spmd/tensor/dispatch.py

```diff
@@ -85,8 +88,27 @@ class OutputSharding:
     failed_reason: Optional[str] = None
 
 
+def is_inplace_op(op_call: torch._ops.OpOverload) -> bool:
+    # util function to check if an op is inplace or not
+    # TODO: bind the core inplace op check to python
```

Comment:
For my learning: what does "bind to python" mean?

Reply:
This is actually the logic I copied from aten C++ code, I think it should be binded to the `_schema` directly instead of we do this manual check here, hence the comment, I'll update with more details. https://github.com/pytorch/pytorch/blob/a8b93cb3ec7cb86a5c9200e8d58b37e9f793e82c/aten/src/ATen/BatchedFallback.cpp#L48